LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2019-2023 The Ethereum Foundation
+Copyright (c) 2019-2024 The Ethereum Foundation
```

Comment:
Is there any drawback to making this 2025 as we're nearing the end of the year here?

Reply:
good idea, done!